Monitoring/munin-murmur.py

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8
 #
 # munin-murmur.py
-# Copyright (c) 2010 - 2016, Natenom <natenom@natenom.com>
+# Copyright (c) 2010 - 2016, Natenom 
```

Comment:
How come there’s a change here in the copyright notice?

You could add yourself here too as you made changes here now.

Reply:
Oh, I didn't want to change that.